wp-cli/media-regenerate.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Helper for dealing with the media regenerate command
+ */
+namespace Automattic\VIP\Commands;
+
+use WP_CLI;
+
+$runner     = WP_CLI::get_runner();
+$assoc_args = $runner->assoc_args;
+
+$media_regenerate_before_invoke = function() use ( &$runner, &$assoc_args ) {
+	// If skip-delete is not set or not true
+	if ( ! isset( $assoc_args['skip-delete'] ) || true !== $assoc_args['skip-delete'] ) {
```

Comment:
We need to check if `true` is a string since WP_CLI doesn't recognize the data type passed into the command options:

```suggestion
	if ( ! isset( $assoc_args['skip-delete'] ) || 'true' !== $assoc_args['skip-delete'] ) {
```

Reply:
Thanks. I fixed in the latest commit.